BUILD.md

```diff
@@ -143,29 +143,67 @@ the number of jobs to use, which should ideally be specified to the number of
 threads your CPU has. You may also want to add that to your preset using the
 `jobs` property, see the [presets documentation][1] for more details.
 
-[1]: https://cmake.org/cmake/help/latest/manual/cmake-presets.7.html
-[2]: https://cmake.org/download/
+## Cross-compile with Docker
 
-## Docker Crossbuilder Image
+The default build architecture is x86_64. You can also cross-compile the distribution-specific Linux package or the
+generic binary with Docker. Both approaches use an x86 (x86_64, amd64) container image to build the artifacts for arm64
+and arm architectures.
 
-The CI job which also runs the included `ziti-builder.sh` builds the `ziti-edge-tunnel` binary inside a Docker
-container. The script will run the necessary container image if needed. The container image has the tools to
-cross-compile for target architectures arm, arm64. This script works for Linux, macOS, and WSL2 on Windows. Arm
-architecture hosts will experience slower build times due to emulation of this x86_64 container image.
+### Build the Linux Package with Docker
 
-Without any arguments, the `ziti-builder.sh` script will build the `bundle` target with the `ci-linux-x64` (amd64)
-preset, placing the resulting ZIP archive in `./build/bundle`.
+The Debian and RedHat packages are built in GitHub and uploaded to DEB and RPM repositories. The Debian package may be
+cross-compiled for arm64 or arm with [a few exceptions](.github/cpack-matrix.yml). Cross-compiling the RPM is not yet
+supported.
+
+1. build the x64 package builder image
+1. run the x64 builder image to build the package for the target architecture
+
+The `ziti-edge-tunnel` binary is also built for the target architecture and included in the package with appropriate
+parameters for the target distribution.
+
+#### Build the Package Builder Image
+
+Build the x64 package builder image for Ubuntu Bionic 18.04. There are builder images for several Ubuntu and RedHat
+vintages that will work with a wide variety of Debian and RPM family distros.
 
 ```bash
-./ziti-builder.sh
+cd ./.github/actions/openziti-tunnel-build-action/ubuntu-22.04/
+docker buildx build --platform linux/amd64 --tag jammy-builder . --load
 ```
 
-To build for a specific target architecture, use the `-p` argument to specify the vcpkg preset.
+#### Run the Package Builder Container
+
+Cross-build the Debian package for arm64 in the x64 builder container. The `ci-linux-arm64` in this example is an
+architecture-specific CMake [preset][1], and the optional TLS library variable overrides the default library, MBed-TLS.
 
 ```bash
-./ziti-builder.sh -p ci-linux-arm64
+docker run \
```

Comment:
does this docker run command require `--platform linux/amd64` if run from an arm64 host?